Week-01/Day-01_Convert-Ages-To-Days/convert_ages_to_days_botiboti/src/main.rs

```diff
@@ -0,0 +1,68 @@
+use num_bigint::{BigUint, ToBigUint};
+use std::io;
+
+// convert input string to BigUint
+fn read_parse(inp: String) -> BigUint {
+    let age: BigUint = inp.trim().parse().expect("Not a positive integer.");
+    age
+}
+
+// turn age in years to days
+fn calc_days(age: BigUint) -> BigUint {
```

Comment:
This function can take the `age` input as a read only reference, so the object doesn't need to be copied on the stack.

Reply:
done

Reply:
I may have been too eager to optimize this one, not sure if it is required here. It makes some things harder (eg. i can't map this function over a `Result<BigUint, e>`. We'll figure this one out with more experience.